### ISSUE TITLE
Semlock Plugin Test randomly fails

### ISSUE THREAD

**Issue by e1528532** [ElektraInitiative/libelektra#1781]
## Steps to Reproduce the Problem

I've noticed that in the elektra-multiconfig-gcc-stable build there is one test that apparently arbitrarily fails or works. Seems to happen in different PRs.

## Expected Result

I expect the test to work always.

## Actual Result

It *sometimes* fails saying

```
ELEKTRA KDB INTERNAL TEST SUITE

Check if script tests the correct version
error: check of plugin semlock failed
error: check of plugin semlock produced: "Was not able to load such a plugin! Maybe you misspelled it, there is no such plugin or the loader has problems. You might want to try to set LD_LIBRARY_PATH, use kdb-full or kdb-static. Errors/Warnings during loading were: Sorry, the error (#145) occurred ;( Description: semlock plugin error Reason: Open semaphore: ENOSYS. /dev/shm should be mounted as tempfs. Please look in the README using "kdb info semlock"! Ingroup: plugin Module: semlock At: /home/jenkins/workspace/workspace/elektra-multiconfig-gcc-stable/build_type/Debug/plugins/ALL/source/src/plugins/semlock/semlock.c:58 Mountpoint: Configfile: Sorry, 2 warnings were issued ;( Warning (#11): Description: open of plugin returned unsuccessfully (reason contains plugin, see other warnings for details) Ingroup: kdb Module: At: /home/jenkins/workspace/workspace/elektra-multiconfig-gcc-stable/build_type/Debug/plugins/ALL/source/src/libs/elektra/plugin.c:294 Reason: semlock Mountpoint: Configfile: Warning (#12): Description: kdbClose() failed Ingroup: kdb Module: At: /home/jenkins/workspace/workspace/elektra-multiconfig-gcc-stable/build_type/Debug/plugins/ALL/source/src/libs/elektra/plugin.c:323 Reason: kdbClose() failed Mountpoint: Configfile: "
testscr_check_kdb_internal_check.sh RESULTS: 176 test(s) done 2 error(s).
```

This happened on different PRs, e.g.
[dini](https://build.libelektra.org/jenkins/job/elektra-multiconfig-gcc-stable/161/build_type=Debug,plugins=ALL/console)
[iowrapper](https://build.libelektra.org/jenkins/job/elektra-multiconfig-gcc-stable/172/build_type=RelWithDebInfo,plugins=ALL/console)
[haskell-cabal-sandboxing](https://build.libelektra.org/jenkins/job/elektra-multiconfig-gcc-stable/build_type=Debug,plugins=ALL/185/console)

which makes me believe its unrelated to my PR and was present before.

## System Information

- whatever the build server uses

**Comment by KurtMi** [comment]
The testcases of the semlock plugin should be deactivated, because there where more problems.

**Comment by KurtMi** [comment]
Well the `testscr_check_kdb_internal_check.sh` does not care if the testcases are deactivated, all added plugins are tested in this testcase. The PR #1798 excludes the `semlock` till properly fixed.

**Comment by markus2330** [comment]
Thanks for the update on this issue!

**Comment by KurtMi** [comment]
closed due to semlock removal #2411
